src/Writers/LaTeXWriter.jl

```diff
@@ -181,6 +201,28 @@ end
 function writeheader(io::IO, doc::Documents.Document)
     custom = joinpath(doc.user.root, doc.user.source, "assets", "custom.sty")
     isfile(custom) ? cp(custom, "custom.sty"; force = true) : touch("custom.sty")
+
+    tag = if haskey(ENV, "CI")
+            if haskey(ENV, "TRAVIS")
+                @info "Travis CI detected"
+                get(ENV, "TRAVIS_TAG",                "")
+            elseif haskey(ENV, "GITLAB_CI")
+                @info "Gitlab CI detected"
+                get(ENV, "CI_COMMIT_TAG",             "")
+            elseif haskey(ENV, "DRONE")
+                @info "Drone CI detected"
+                get(ENV, "DRONE_TAG",                 "")
+            elseif haskey(ENV, "CIRRUS_CI")
+                @info "Cirrus CI detected"
+                get(ENV, "CIRRUS_TAG",                "")
+            elseif haskey(ENV, "APPVEYOR")
+                @info "AppVeyor CI detected"
+                get(ENV, "APPVEYOR_REPO_TAG_NAME",    "")
+            end
+            @info "No tag detected..."
+            else
+                ""
+            end
```

Comment:
The indentation is a bit broken here.

But in any case -- this actually duplicates the code above. Could we refactor the `if` into a function that we can re-use here and above?

I am also wondering if it might be better to make the `@info`s `@debug`s. Although, there might be value in being verbose about this automagic behavior here.

Reply:
I can make it a function, will take a while though - dont have too much free time today, so maybe in the evening.